src/AutoPilotPlugins/PX4/RadioComponent.cc

```diff
@@ -140,7 +140,11 @@ QStringList RadioComponent::paramFilterList(void) const
 
 QWidget* RadioComponent::setupWidget(void) const
 {
-    return new PX4RCCalibration;
+    QGCQmlWidgetHolder* holder = new QGCQmlWidgetHolder();
```

Comment:
@DonLakeFlyer may i ask you, why you don't use C++11 `auto`? I think that is very handy, and i widely use it in mavros.
Only one thing, seems that compiling is slower, over one minute for such a small project.